fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/PermissionReadException.java

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.fiat.permissions;
+
+/**
+ * An exception reading permission values.
+ *
+ * <p>Reading permissions is side-effect free so this exception is retryable by default.
+ */
+public class PermissionReadException extends PermissionRepositoryException {
+  public PermissionReadException(String message) {
+    super(message);
+    setRetryable(true);
```

Comment:
So is the idea that the `retryable` field in this exception would bubble up to the caller and we could retry somewhere else (like maybe a retry across the wire?).

Reply:
Cool, gotcha.